application/src/main/java/org/thingsboard/server/controller/BaseController.java

```diff
@@ -279,11 +282,37 @@
     @Getter
     protected boolean edgesEnabled;
 
+    @ExceptionHandler(Exception.class)
```

Comment:
What is the motivation for this?

Reply:
To avoid wrapping each controller's API method in a try-catch block that maps Exception to ThingsboardException.